backend/app/views/spree/admin/orders/customer_details/edit.html.erb

```diff
@@ -1,11 +1,10 @@
 <%= render :partial => 'spree/admin/shared/order_tabs', :locals => { :current => 'Customer Details' } %>
 
 <% content_for :page_title do %>
-  <i class="fa fa-arrow-right"></i> <%= Spree.t(:customer_details) %>
+  &#x2f; <%= Spree.t(:customer_details) %>
 <% end %>
 
 <% content_for :page_actions do %>
```

Comment:
Something along those lines was my thought when I was going through but in hindsight I suppose since they're just empty `content_for`s with no text in them they wouldn't even affect the rendering of that element on the page or not.  I'm not sure if that will cause the `page_actions` block to still render on the page but I imagine we would want to keep it for the potential deface overrides.